pages.it/common/vlc.md

```diff
@@ -0,0 +1,24 @@
+# vlc
+
+> Lettore multimediale multi-piattaforma.
+> Maggiori informazioni: <https://www.videolan.org/vlc/>.
+
+- Riproduce un file:
+
+`vlc {{percorso/al/file}}`
+
+- Riproduce a schermo intero:
+
+`vlc --fullscreen {{percorso/al/file}}`
+
+- Riproduce mutato:
+
+`vlc --no-audio {{percorso/al/file}}`
+
+- Riproduce ciclicamente:
```

Comment:
Ciclicamente means periodically. I'd suggest using "Riproduce in ciclo continuo"